azext_edge/edge/providers/support/base.py

```diff
@@ -480,3 +496,10 @@ def _process_kubernetes_resources(
         )
 
     return processed
+
+
+def exclude_resources_with_prefix(resources: List[dict], exclude_prefixes: List[str]) -> List[str]:
```

Comment:
This doesn't return List[str] does it. Also, resources, isn't being passed List[dict] right now.

Reply:
yeah, thanks!